docs/variables/customapi.md

```diff
@@ -4,41 +4,61 @@ id: customapi
+| :-----------------------------: | :----------------------------------------------------------------------------------: |
+|     `x-fossabot-channelid`      |                   The internal **Fossabot ID** of a broadcaster.                     |
+| `x-fossabot-channeldisplayname` |   The uppercase version, or internationalized version of a broadcaster's username.   |
+|    `x-fossabot-channellogin`    |   The lowercase version of a broadcaster's username (otherwise known as *login*).    |
+|    `x-fossabot-channelslug`     | The **Fossabot channel URL** of a broadcaster. Useful for linking to commands pages. |
+|  `x-fossabot-channelprovider`   |               The platform the request was sent from (e.g. `twitch`).                |
+| `x-fossabot-channelproviderid`  |                The platform ID (e.g. the Twitch ID) of a broadcaster.                |
+|     `x-fossabot-hasmessage`     | `true` or `false` depending on whether a message sent this (`false` if timer, etc).  |
+|          `user-agent`           |                                `Fossabot Web Proxy`                                  |
+
+### List of Message Headers
+
+These headers are present in requests **sent from messages** and therefore attached to a particular user.
 
 |                 Name                 |                                                                    Value                                                                     |
 | :----------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------: |
-|       `x-fossabot-message-id`        |                                                 The message ID from the provider, eg. Twitch                                                 |
-|    `x-fossabot-message-userlogin`    |                                         The lowercase username of the sender (also known as "login")                                         |
-| `x-fossabot-message-userdisplayname` |                                 The uppercased version, or internationalized version of the sender username                                  |
-|  `x-fossabot-message-userprovider`   | The platform the message was sent from by the user (this is useful if a discord user toggles a twitch command through the discord bot, etc). |
-| `x-fossabot-message-userproviderid`  |                                                   The senders' platform ID, ie: Twitch ID                                                    |
+|       `x-fossabot-message-id`        |                                                 The message ID from the provider (e.g. Twitch).                                              |
+|    `x-fossabot-message-userlogin`    |                                The lowercase version of the sender's username (otherwise known as *login*).                                  |
```

Comment:
apostophe flag again, these are prob annoying just mark these as resolved if im wrong

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/18#discussion_r786576213).